<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteManager</class>
 <widget class="QDialog" name="RemoteManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Host Manager</string>
  </property>
  <widget class="RDTreeWidget" name="hosts">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <attribute name="headerStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Hostname</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Running</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="progressIcon">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Resources/resources.qrc">:/hourglass.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="progressText">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote connections in progress. Please wait...</string>
   </property>
  </widget>
  <widget class="QLabel" name="progressCount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>999 connections active</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>381</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottomLayout">
    <item>
     <widget class="QGroupBox" name="configGroup">
      <property name="title">
       <string>Host configuration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="hostnameLabel">
         <property name="text">
          <string>Hostname:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RDLineEdit" name="hostname"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="addUpdateHost">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="runCommandLabel">
         <property name="text">
          <string>Run Command: Configure a command to run that launches the remote server on this host.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="RDLineEdit" name="runCommand"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="opsGroup">
      <property name="title">
       <string>Operations</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="refreshOne">
         <property name="text">
          <string>Refresh Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshAll">
         <property name="text">
          <string>Refresh All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>Connect to App</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteHost">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RDTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>Widgets/Extended/RDTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>RDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/Extended/RDLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
